charts-external/openpension/templates/deployment-client.yaml

```diff
@@ -0,0 +1,34 @@
+{{ if .Values.enabled }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: client
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  strategy:
+    type: Recreate
```

Comment:
for the client you can remove this strategy - to use the default rollout strategy (assuming multiple clients can run in parallel)